composer.json

```diff
@@ -25,7 +27,10 @@
         "ext-mbstring": "*",
         "ext-curl": "*",
         "composer/ca-bundle": "~1.0",
-        "symfony/options-resolver": "~2.7|~3.0"
+        "symfony/options-resolver": "~2.7|~3.0",
+        "php-http/httplug": "~1.1",
+        "php-http/curl-client": "~1.7",
```

Comment:
We used to give a one liner to install the package without any problems (https://github.com/fbourigault/php-gitlab-api/blob/e49bc7313ae53e26501993353dcc1d404ae26b01/README.md#installation)